### PR TITLE
En 1366 fix edit label (partial PR)

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/labels/common/LabelsForm.js [hunk 1]
```diff
@@ -7,13 +7,12 @@ import { FormattedMessage } from 'react-intl';
...
 
-
 const EDIT_MODE = 'edit';
 const NEW_MODE = 'new';
 
-export class AddLabelsPageFormBody extends Component {
+export class LabelsFormBody extends Component {
   componentWillMount() {
-    this.props.onWillMount();
+    this.props.onWillMount(this.props);
```
Comment (nicpuddu) [inline src/ui/labels/common/LabelsForm.js]: why are you passing all the props to this on will mount?

File: test/ui/labels/add/AddFormContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import 'test/enzyme-init';
+
+import { mapStateToProps, mapDispatchToProps } from 'ui/labels/add/AddFormContainer';
+
+const dispatchMock = jest.fn();
+const TEST_STATE = {
+  languages: {},
+  labels: {},
+};
```
Comment (nicpuddu) [inline test/ui/labels/add/AddFormContainer.test.js]: no point on having a test state if you are mocking the selectors

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**